includes/templates/reader-activation-emails/otp.php

```diff
@@ -18,39 +18,51 @@
+	<div class="wp-block-button"><a class="wp-block-button__link has-' . esc_attr( $primary_text_color ) . '-color has-text-color has-background has-link-color has-text-align-center wp-element-button" href="*MAGIC_LINK_URL*" style="border-radius:4px;background-color:' . esc_attr( $primary_color ) . '">' . sprintf( /* Translators: s is the site's title. */ __( 'Continue to %s', 'newspack' ), '*SITE_TITLE*' ) . '</a></div>
 	<!-- /wp:button --></div>
 	<!-- /wp:buttons -->
 	<!-- wp:paragraph -->
-	<p>' . __( 'Or copy and paste the link in your browser:', 'newspack' ) . ' <a href="*MAGIC_LINK_URL*" data-type="URL" data-id="*MAGIC_LINK_URL*">*MAGIC_LINK_URL*</a></p>
+	<p>' . __( 'Or copy and paste the link in your browser:', 'newspack' ) . ' <br><a href="*MAGIC_LINK_URL*" data-type="URL" data-id="*MAGIC_LINK_URL*">*MAGIC_LINK_URL*</a></p>
 	<!-- /wp:paragraph -->
 	<!-- wp:paragraph -->
 	<p>' . __( 'If you did not request this code, please ignore this email.', 'newspack' ) . '</p>
 	<!-- /wp:paragraph --></div>
 	<!-- /wp:group -->' .
-
+	// FAQ.
+	'<!-- wp:group {"style":{"spacing":{"padding":{"top":"56px","bottom":"56px","left":"56px","right":"56px"}},"background":"' . esc_attr( $secondary_color ) . '","color":{"text":"' . esc_attr( $secondary_text_color ) . '","background":"' . esc_attr( $secondary_color ) . '"},"elements":{"link":{"color":{"text":"' . esc_attr( $primary_color ) . '"}}}},"className":"has-secondary-background-color has-background","layout":{"type":"constrained"}} -->
+	<div class="wp-block-group has-secondary-background-color has-background has-text-color has-link-color" style="color:' . esc_attr( $secondary_text_color ) . ';background-color:' . esc_attr( $secondary_color ) . ';padding-top:56px;padding-right:56px;padding-bottom:56px;padding-left:56px">
+	<!-- wp:heading {"level":3,"style":{"typography":{"fontStyle":"normal","fontWeight":"400"}}} -->
+	<h3 class="wp-block-heading" style="font-style:normal;font-weight:400">' . __( 'Frequently Asked Questions', 'newspack' ) . '</h3>
+	<!-- /wp:heading -->
+	<!-- wp:paragraph -->
+	<p><strong>' . __( 'What is a magic link?', 'newspack' ) . '</strong><br>' . __( 'It\’s a temporary, password-free link like the one above to help you quickly and securely verify your identity and sign in – all without needing to set a password.', 'newspack' ) . '</p>
+	<!-- /wp:paragraph -->
+	<!-- wp:paragraph -->
+	<p><strong>' . __( 'Can I still create a password?', 'newspack' ) . '</strong><br>' . sprintf( /* Translators: 1: opening HTML anchor tag 2: closing HTML anchor tag */ __( 'Yes. For security reasons, we recommend signing in with Google, Facebook, or Apple, or using a one-time code sent to your email, but you can set a password if you prefer. %1$sClick here%2$s to create a password.', 'newspack' ), '<a href="*SET_PASSWORD_LINK*" data-type="link" data-id="*SET_PASSWORD_LINK*">', '</a>' ) . '</p>
```

Comment:
We don't support signing in with Facebook and Apple yet. This should only mention Google.

Reply:
Good catch! Removed in https://github.com/Automattic/newspack-plugin/pull/3034/commits/51015903b08ab45e69982a34997e127c696fa3bd